tutorials/2d/2d_lights_and_shadows.rst

```diff
@@ -6,238 +6,324 @@
-
-When you have added the :ref:`LightOccluder2Ds <class_LightOccluder2D>` the shadows still won't
-appear. You need to go back into the :ref:`PointLight2Ds <class_PointLight2D>` and under the Shadow
-section set ``Enable`` to ``on``. This turns on shadows with hard edges like in the image below.
-
-.. image:: img/light_shadow_filter0_pcf0.png
-
-To give the shadows that nice, soft edge look we set the variables ``filter``, ``filter smooth``, and
-``gradient length``. Godot supports `Percentage Closer Filtering <https://developer.nvidia.com/gpugems/GPUGems/gpugems_ch11.html>`_
-(PCF), which takes multiple samples of the shadow map around a pixel and blurs them to create
-a smooth shadow effect. The higher the number of samples the smoother the shadow will
-look, but the slower it will run. That is why Godot provides 3-13 samples by default and allows you to choose.
-The demo uses PCF7.
-
-.. image:: img/light_shadow_normal.png
-
-.. note:: This is a shadow rendered with the demo's settings. ``gradient length`` is set
-          to ``1.3``, ``filter smooth`` is set to ``11.1``, and ``filter`` is set to ``PCF7``.
-
-.. image:: img/light_shadow_pcf13.png
-
-.. note:: ``filter`` is set to ``PCF13``. Notice how the shadow becomes wider, this is because the
-          distance between samples is based on the variable ``filter smooth``.
-
-In order to make use of filtering you need to set the ``filter smooth`` variable.
-This dictates how far apart the samples are. If you want the soft area to extend quite far, you can increase
-the size of ``filter smooth``. However, with few samples and a large filter smooth, you can see lines
-forming between the samples.
-
-.. image:: img/light_shadow_filter30.png
-
-.. note:: ``filter smooth`` is set to ``30``.
-
-The different :ref:`Light <class_Light2D>` nodes in the demo use different values for filter smooth.
-Play around with it and see what you like.
-
-.. image:: img/light_shadow_filter0.png
-
-.. note:: ``filter smooth`` is set to ``0``.
-
-Lastly, there is the variable ``gradient length``. For some smooth shadows it is preferable not to have the
-shadow start immediately on the object, as this produces a hard edge. The gradient length variable creates
-a smooth gradient to begin the shadow to reduce the effect of the hard edge.
-
-.. image:: img/light_shadow_grad0.png
-
-.. note:: ``gradient length`` is set to ``0``.
-
-.. image:: img/light_shadow_grad10.png
-
-.. note:: ``gradient length`` is set to ``10``.
-
-You will need to play around with the options a bit to find settings that suit your project. There is no right solution
-for everyone, which is why Godot provides so much flexibility. Just keep in mind that the higher ``filter``
-set the more expensive the shadows will be.
+Also known as omnidirectional (omni) or spot lights, point lights are the most
+common element in 2D lighting. Point lights can be used to represent light from
+torches, fire, projectiles, etc.
```

Comment:
```suggestion
Point lights are the most common element in 2D lighting. Point lights can be used
to represent light from torches, fire, projectiles, etc.
```

We have discussed the parallels to 3D lights before and agreed that PointLights are their own thing. Calling them omni lights or spot lights is not accurate.